dlib/image_transforms/interpolation.h

```diff
@@ -310,7 +310,9 @@ namespace dlib
                     pixel_to_vector<double>(img[r+1][c  ])(i),
                     pixel_to_vector<double>(img[r+1][c+1])(i));
             typename image_view_type::pixel_type temp;
-            vector_to_pixel(temp, pvout);
+            const auto min_val = pixel_traits<pixel_type_t<image_view_type>>::min();
+            const auto max_val = pixel_traits<pixel_type_t<image_view_type>>::max();
```

Comment:
I highly doubt this would have made any difference at all since the compiler would have probs inlined them, but `min_val` and `max_val` could have been `constexpr`.

Reply:
Yeah I wouldn't worry about it.  `constexpr` is useful for compile time stuff (e.g. doing template magic).  It shouldn't make a difference to things like this.